core/src/main/java/nl/nn/adapterframework/receivers/ExchangeMailListener.java

```diff
@@ -131,7 +131,7 @@ public Message extractMessage(Item rawMessage, Map<String,Object> threadContext)
 
 	private void addEmailInfoSimple(EmailMessage emailMessage, XmlBuilder emailXml) throws Exception {
 		XmlBuilder subjectXml = new XmlBuilder("subject");
-		subjectXml.setCdataValue(emailMessage.getSubject());
+		subjectXml.setValue(emailMessage.getSubject());
```

Comment:
Wanneer moeten we setCdataValue nu wel gebruiken? Kan die helemaal weg als setValue het zelfde doet?